.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Stayed on weekly because it used to be monthly. I do not really see the point of not having it `daily`. Is there a shortcoming?